hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/recon/TestReconTasks.java

```diff
@@ -166,5 +179,64 @@ public void testMissingContainerDownNode() throws Exception {
                   0, 1000);
       return (allMissingContainers.isEmpty());
     });
+    IOUtils.closeQuietly(client);
+  }
+
+  @Test
+  public void testEmptyMissingContainerDownNode() throws Exception {
```

Comment:
Can we add a comment (short paragraph) describing the overall purpose and flow of this integration test?

Reply:
Done.

Reply:
Thanks a lot @devmadhuu !